internal/plugins/ansible/v1/init.go

```diff
@@ -174,3 +149,84 @@ func (p *initSubcommand) PostScaffold() error {
 
 	return nil
 }
+
+// addInitCustomizations will perform the required customizations for this plugin on the common base
+func addInitCustomizations(projectName string) error {
+	managerFile := filepath.Join("config", "manager", "manager.yaml")
+
+	// todo: we ought to use afero instead. Replace this methods to insert/update
+	// by https://github.com/kubernetes-sigs/kubebuilder/pull/2119
+
+	// Add leader election
+	err := sdkutil.InsertCode(managerFile,
+		"--leader-elect",
+		fmt.Sprintf("\n        - --leader-election-id=%s", projectName))
+	if err != nil {
+		return err
+	}
+	err = sdkutil.InsertCode("config/default/manager_auth_proxy_patch.yaml",
+		"- \"--leader-elect\"",
+		fmt.Sprintf("\n        - \"--leader-election-id=%s\"", projectName))
+	if err != nil {
+		return err
+	}
+
+	// remove the resources limits
+	// todo: remove it when we solve the issue operator-framework/operator-sdk#3573
+	const resourcesLimitsFragment = `  resources:
+          limits:
+            cpu: 100m
+            memory: 30Mi
+          requests:
+            cpu: 100m
+            memory: 20Mi
+      `
+	err = sdkutil.ReplaceInFile(managerFile, resourcesLimitsFragment, "")
+	if err != nil {
+		return err
+	}
+
+	// Add ANSIBLE_GATHERING env var
+	const envVar = `
+        env:
+          - name: ANSIBLE_GATHERING
+            value: explicit`
```

Comment:
Small nit to align YAML formatting
```suggestion
        env:
        - name: ANSIBLE_GATHERING
          value: explicit`
```

Reply:
done